### PR TITLE
Feature/4160 add provider to i18n instance

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # BLUI-4160

#### Changes proposed in this Pull Request:

- Added i18n instance
- Added translations
- Added i18n instance provider

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- Unable to test unless we have component

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by daileytj** [comment]
If you can't test it, how do you know this is going to work? 

We are not going to have one shared instance either. You will need to make a separate instance within the context of your AuthContextProvider. You will also need to pair down those translation files to only include the items needed for the AuthContextProvider contents.

**Comment by surajeaton** [comment]
Converted to draft, I will be using the example from #266 . Will open this once that PR gets merged.

File: login-workflow/src/new-architecture/i18nInstance/index.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    .createInstance(
+        {
+            lng: 'en',
+            detection: {
+                order: ['querystring', 'localStorage', 'navigator'],
+                caches: ['localStorage'],
+                lookupLocalStorage: 'blui-auth-i18nextLng',
+            },
+            fallbackLng: 'en',
+            ns: ['blui', 'app'],
```
Comment (joebochill) [inline login-workflow/src/new-architecture/i18nInstance/index.ts:16]: Refer to tom's implementation on the Registration side...there should only be a single instance and it should be something like bluiAuthentication

File: login-workflow/src/new-architecture/i18nInstance/translations/chinese.ts [hunk 2]
```diff
@@ -0,0 +1,182 @@
+import { LanguageFile } from './types';
+
+const resources: LanguageFile = {
+    translation: {
+        ACTIONS: {
```
Comment (joebochill) [inline login-workflow/src/new-architecture/i18nInstance/translations/chinese.ts:5]: You've got a lot of keys/strings in here that are not part of the Auth screens

File: login-workflow/src/new-architecture/i18nInstance/provider.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ * @packageDocumentation
+ * @module I18nInstanceProvider
+ */
+
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { i18nBluiInstance } from '.';
+import { I18nInstanceProviderProps } from './types';
+
+export const I18nInstanceProvider: React.FC<React.PropsWithChildren<I18nInstanceProviderProps>> = (props) => {
```
Comment (joebochill) [inline login-workflow/src/new-architecture/i18nInstance/provider.tsx:11]: Not sure why we have a separate component for this. The `<I18nextProvider>` should just be the first thing returned in the AuthContextProvider...we shouldn't need an extra wrapper component.

**Comment by surajeaton** [comment]
Closing this PR, this will be covered #271

**Pull request closed without merging**